CONTRIBUTING.md

```diff
@@ -18,7 +18,7 @@ To learn all about contributing to the Gutenberg project, see the [Contributor G
 
 -   As with all WordPress projects, we want to ensure a welcoming environment for everyone. With that in mind, all contributors are expected to follow our [Code of Conduct](/CODE_OF_CONDUCT.md).
 
--   All WordPress projects are [licensed under the GPLv2+](/LICENSE.md), and all contributions to Gutenberg will be released under the GPLv2+ license. You maintain copyright over any contribution you make, and by submitting a pull request, you are agreeing to release that contribution under the GPLv2+ license.
+-   You maintain copyright over any contribution you make. By submitting a pull request you agree to release that code under both the GPLv2+ and MPL licenses as stated in [Gutenberg's License](/LICENSE.md).
```

Comment:
I wonder if it's suitable to actually not mention the license but just reference the license file? That way the license itself will only be mentioned once and the ref won't easily get out of date.

For example:
```suggestion
-   You maintain copyright over any contribution you make. By submitting a pull request you agree to release that code under the license stated in [Gutenberg's License](/LICENSE.md).
```
WDYT?

Reply:
I like it.